frontend/pages/queries/ManageQueriesPage/ManageQueriesPage.tsx

```diff
@@ -165,18 +163,6 @@ const ManageQueriesPage = ({
     }
   }, [refetchFleetQueries, selectedQueryIds, toggleDeleteQueryModal]);
 
-  const renderPlatformDropdown = () => {
```

Comment:
since this is only used in TableContainer, moved this closer to code (moved to QueriesTable.tsx which contains TableContainer). More componentized and follows pattern of host details cards where dropdown renders live on cards not on page level.